File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.js
function solution(n) {
    var answer = "";
    
    // 10진수 -> 3진수 변환
    while (n > 0) { // n이 1보다 클 동안 반복해주면서
        answer += (n % 3).toString(); // 임시 문자변수에 n을 3으로 나눈 나머지값을 answer에 더해줌
        n = Math.floor(n / 3); // n의 값은 3으로 나눈 몫으로 업데이트 해줌
    }
    
    return parseInt(answer, 3); // 3진수로 쓰여진 숫자를 10진수로 변환하여 반환
}

File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.py
def solution(n):
    temp = "" # 임시변수 선언
    
    # 10진수 -> 3진수 변환과정
    while n > 0:
        temp = str(n % 3) + temp
        n //= 3
        
    # 3진수 문자열을 뒤집기
    reversed_temp = temp[::-1]
    
    # 뒤집은 문자열을 다시 10진수로 변경 후 반환
    return int(reversed_temp, 3)